lib/execution_engine2/utils/Condor.py

```diff
@@ -113,89 +87,6 @@ def _setup_environment_vars(self, params: Dict, client_group: str) -> str:
 
         return f'"{environment}"'
 
-    @staticmethod
-    def _check_for_missing_runjob_params(params: Dict[str, str]) -> None:
```

Comment:
`JobSubmissionParameters` class is passed in to `Condor.run_job` so the parameters are already validated